docs/concepts.rst

```diff
@@ -0,0 +1,61 @@
+.. comment:
```

Comment:
Nice to get the documentation working again, and nice overview of what's openstef!